ssnm/templates/map_page.html

```diff
@@ -1,31 +1,33 @@
 {% extends "base.html" %}
 {% load bootstrap %}
-{% load static from staticfiles %}
+{% load compress %}
```

Comment:
Did you mean to add this here? It looks like you're not using django-compressor on this page.